.github/workflows/release.yaml

```diff
@@ -19,8 +19,8 @@ jobs:
       with:
         virtualenvs-create: true
         virtualenvs-in-project: true
-    - name: Install library and dependencies
-      run: poetry install --all-extras --only test --no-interaction
+    - name: Install library and test dependencies
+      run: poetry install --all-extras --only main,test --no-interaction
```

Comment:
I learned that with poetry `--only` needs to be followed by `main` if you want default package dependencies to be installed.